hieradata/common.yaml

```diff
@@ -246,6 +246,9 @@ govuk::apps::bouncer::db_hostname: "transition-postgresql-slave-1.backend"
 govuk::apps::bouncer::nagios_memory_warning: 1400
 govuk::apps::bouncer::nagios_memory_critical: 1500
 
+govuk::apps::collections_publisher::redis_host: 'redis-1.backend'
```

Comment:
Yes, this should be hard coded as we need to use redis-1 specifically. Other apps already specify redis-1 too.